### DISCUSSION TITLE
Common Columns in Multiple Models

### DISCUSSION THREAD

**Discussion by stealthco-tony** [pallets-eco/flask-sqlalchemy#1306] [Q&A]
In our database, we will need several fields that are common in each table. Things like "created_date", "last_updated_date", "is_active", "approved". We will have several hundred tables in our database. How can we easily create a "common_model.py" file and then import / merge all those fields with the fields specific to each table?

Thanks!

**Comment by davidism** [comment]
This is better answered as a question to sqlalchemy, flask-sqlalchemy doesn't affect it. Search their docs for "abstract" or "mixin".

**Reply by stealthco-tony** [reply]
Thanks @davidism !
